.travis.yml

```diff
@@ -0,0 +1,33 @@
+# vim ft=yaml
+
+# After changing this file, check it on:
+#   http://yaml-online-parser.appspot.com/
+
+# See doc/travis_notes.txt for some guidelines
+
+language: python
+
+sudo: required
+
+python:
+  - 2.7
+  - 3.5
+
+# See http://docs.travis-ci.com/user/caching/#pip-cache
+cache: pip
+
+before_install:
+  - sudo apt-get install python-vtk
```

Comment:
Why is python-vtk required?

Reply:
it was an attempt to run `mayavi` in CI but I did not succeed, so also the functions with `mayavi` are not part of tests but samples only...

Reply:
Ah, I see. The code in `tests.py` is meant to show the users how to use the library, not to serve as a proper code test. I'll add a `test_morphsnakes.py` with unit tests (as [the one I wrote](https://github.com/pmneila/scikit-image/blob/8e918e6c101aa980b50865b07b67e7f1ef34894f/skimage/segmentation/tests/test_morphsnakes.py) for `scikit-image`). `mayavi` shouldn't be required for testing the library.

Maybe I should also rename `tests.py` to `examples.py` to avoid confusions.

Reply:
well, it may depend if the result is stochastic or not, but anyway, this test/example is a kind of verification that the whole process passed without failer...
regarding the examples, I would prefer ipynb where you have clear association between code and particular results, e.g. [sample_lakes](https://github.com/Borda/morph-snakes/blob/master/notebooks/sample_lakes.ipynb)

Reply:
Yes, but with pytest (or unittest) we want to test not only good examples, but also possible errors on user input (shape mismatch, incorrect data types, empty levelset...). I don't want to include those as toy examples in the documentation.

Reply:
well, it is up to you... I think those good examples may be as doctest and the error testing as unit tests